code/modules/economy/TradeDestinations.dm

```diff
@@ -16,85 +16,73 @@ var/list/weighted_mundaneevent_locations = list()
 /datum/trade_destination/proc/get_custom_eventstring(var/event_type)
 	return null
 
-//distance is measured in AU and co-relates to travel time
-/datum/trade_destination/centcomm
-	name = "CentCom"
-	description = "NanoTrasen's administrative centre for Tau Ceti."
+//distance is measured in Arbitrary and corelates to travel time, like, I guess
```

Comment:
correlates, two Rs

Reply:
not fixing typos in comments on a file that's nigh-obsolete anyway